interfaces/Referral-App/src/app/referral/referral.page.ts

```diff
@@ -17,7 +17,10 @@ import { ReferralPageDataService } from 'src/app/services/referral-page-data.ser
   templateUrl: 'referral.page.html',
   styleUrls: ['referral.page.scss'],
 })
-export class ReferralPage {
+export class ReferralPage implements OnInit {
+  public region: string;
+  public supportedRegions: string[] = ['amsterdam', 'utrecht'];
```

Comment:
There isn't really a more convenient place 'higher up' then this, right?

As I feel it should be defined somewhere in the `environment`-file, but we would have the 'hard' connection to these 2 strings anyway in other places...

Reply:
Do env vars support JSON values? Then we can move these there.

Reply:
No, not really...

We use a "comma-separated-list" for the LOCALES:
https://github.com/global-121/121-platform/blob/master/interfaces/PA-App/src/app/personal-components/select-language/select-language.component.ts#L46

So the ENV would be: `SUPPORTED_REGIONS=amsterdam,utrecht`; And you could use that as an array somewhere in the code then...

But that would make it actually more complex and harder to understand, I think.
As there needs to be a 'hard' connection between the SpreadsheetId and the URL-path...

But putting a 'real' JSON-object like this in an ENV-var, feels weird:
```
REGIONS={ "amsterdam": "abcDeFGhI", "utrecht": "JklMnOPqRst" }
```